### PR TITLE
Capitalization edits from @dwahbe

### PR DESCRIPTION
**Pull request by garyhtou**

### CODE REVIEW

File: app/views/users/_settings_nav.html.erb [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <% end %>
   <%= settings_tab active: active == "previews" do %>
     <%= link_to @user == current_user ? settings_previews_path : previews_user_path(@user), data: { turbo: true, turbo_action: "advance" } do %>
-      <span class="xs-hide">Feature&nbsp;</span>Previews
```
Comment (sampoder) [inline app/views/users/_settings_nav.html.erb]: This has been on my list to fix for a while but on mobile this will show up as "previews"

Comment (garyhtou) [reply app/views/users/_settings_nav.html.erb]: Ah good point

Comment (garyhtou) [reply app/views/users/_settings_nav.html.erb]: Maybe we can do some css magic

Comment (garyhtou) [reply app/views/users/_settings_nav.html.erb]: Or, just show the full "Feature previews" on mobile. I'm not sure how strong the need is to truncate

**Pull request merged**